lib/charms/ovn_charm.py

```diff
@@ -765,6 +777,9 @@ def configure_ovs_dpdk(self):
             ('dpdk-extra',
              self.dpdk_eal_allow_devices(dpdk_context.devices())
              if self.options.enable_dpdk else None),
+            ('pmd-cpu-mask',
+             dpdk_context.pmd_cpu_mask()
+             if self.options.enable_dpdk else None),
```

Comment:
This does not check for whether the charm `pmd-cpu-mask` option is set, and leads to putting `pmd-cpu-mask="0x0"` into the Open vSwitch database when unset.

I'd rather we refrain from putting the option into the database when charm config is unset so that we can use what Open vSwitch does by default, which may develop over time.

I guess just extending the check with `and self.options.pmd_cpu_mask` would be sufficient?

Reply:
ok indeed! Will make the change.

Reply:
done